useragent/parse.go

```diff
@@ -265,23 +265,23 @@ func requireWhitespace(data []byte, from int) (next int, ok bool) {
 	return next, true
 }
 
-// tchar           = "!" / "#" / "$" / "%" / "&" / "'" / "*"
-//                 / "+" / "-" / "." / "^" / "_" / "`" / "|" / "~"
-//                 / DIGIT / ALPHA
-//                 ; any VCHAR, except delimiters
+// tchar           = "!" / "#" / "$" / "%" / "&" / "'" / "*",
+//                 / "+" / "-" / "." / "^" / "_" / "`" / "|" / "~",
+//                 / DIGIT / ALPHA,
+//                 ; any VCHAR, except delimiters.
```

Comment:
These changes look wrong, because `,` has different meaning in bdnf, maybe it'll accept:
```
//   tchar         = "!" / "#" / "$" / "%" / "&" / "'" / "*"
//                 / "+" / "-" / "." / "^" / "_" / "`" / "|" / "~"
//                 / DIGIT / ALPHA
//                 ; any VCHAR, except delimiters
```